app/src/App.tsx

```diff
@@ -59,9 +59,7 @@ const App: React.FC = (props: any) => {
           >
             <ContentWrapper>
               <ConnectionModalNavigation>
-                <ModalTitle>
-                  <span style={{ fontFamily: 'Roboto' }}>Settings</span>
-                </ModalTitle>
+                <ModalTitle marginTop={'2px'}>Settings</ModalTitle>
```

Comment:
What is the use of this top margin? Seems unrelated to the issue.... was it wrong?

Reply:
@Mi-Lan the marginTop of 2px was there before as in the case of the comment below. 

It aligns it vertically, if you delete it, you see it's not 100% vertically aligned. It seems to me it's aligned with the marginTop, that's why I kept it

Reply:
@Frankaus Thats not proper way to align items vertically....Look at other examples in the codebase how we do it...

Reply:
Agreed